src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -128,6 +124,22 @@ const SidebarLinks = (props) => {
             ]}
             >
                 <Header textSize="large" title="Chats" />
+                <View style={[chatSwitcherStyle]}>
```

Comment:
The `View` with `chatSwitcherStyle` needs to go back where it was before. Only the `Touchable` + `MagnifyingGlassIcon` should be in between the `Header` and `AvatarWithIndicator`.

We don't need to explicitly hide the search icon in this case because it will be covered by the `ChatSwitcherView` once it renders. Apologies if this deviates from the original instructions slightly.

Reply:
Updated